packages/ckeditor5-coremedia-richtext/src/CoreMediaRichTextConfig.ts

```diff
@@ -134,13 +134,30 @@ const defaultRules: FilterRuleSetConfiguration = {
     ...tableRules,
     span: langMapperConfiguration,
     pre: langMapperConfiguration,
-    "xdiff:span": (params) => {
```

Comment:
As we prevent saving XDiff augmented markup to the server anyway, there is no need for this extra clean-up on `toData` processing. It was incomplete anyway for previous CKEditor 4 solution, where we missed to (explicitly) strip the xdiff-attribute from `<img>` elements.